R/agg_scale.R

```diff
@@ -35,6 +35,11 @@
 #'   Whether to drop aggregates (or overlapping intervals) that are already
 #'   present in `dt` before aggregating. Default is 'False' and the function
 #'   errors out.
+#' @param collapse_interval_cols \[`logical(1)`\]\cr
+#'   Whether to collapse interval `id_cols` (not including `col_stem` if it is
```

Comment:
This description is a bit confusing to me.

Reply:
I'm planning to add a vignette related to scaling that will provide a better example of this, does it make more sense in the details section?

Reply:
Yes, the details section helps, and so would a vignette. Seems fine to me then.